src/Core/Domain/CmsPageCategory/Command/AbstractCmsPageCategoryCommand.php

```diff
@@ -47,12 +44,12 @@ abstract class AbstractCmsPageCategoryCommand
     protected function assertCategoryName(array $names)
     {
         foreach ($names as $name) {
-            if (!preg_match(self::CATEGORY_NAME_REGEX_PATTERN, $name)) {
+            if (!preg_match(RegexPattern::CATALOG_NAME, $name)) {
```

Comment:
Why rename CATEGORY_NAME into CATALOG_NAME?
A catalog is different than a category. More we're in Category context.

Reply:
It is the same regex pattern. It was validated by isCatalogName in legacy before and the category belongs to catalog context, so I thought why not :thinking: . If this is not suitable, then this PR is might be not needed at all, as all duplicate regex patterns could be considered just coincidental and each be kept in corresponding domain without trying to reuse ?

Reply:
Even if it's the same, the context is different.
A catalog is a list of items, and a category describe what kind of item is it.
If in the future, we need to update this regex pattern only for one case like we did for the Customer name, we're fucked!
We must separate regex, or otherwise, use a more generic pattern, or something stupid as `CATALOG_OR_CATEGORY_NAME`

Reply:
Ok then this PR seems to be a bad idea.

Reply:
Eyyyyyh 😄 dont go so fast
The PR is a good idea for generic names, not specific names

Reply:
The idea behind this PR was good imo.

Reply:
All of currently defined patterns in RegexPattern.php are generic ones, specific are kept in corresponding domain models, e.g. IsoCode or TagIETF. Category name isn't specific, its the same catalog_name :confused: . Idk maybe its confusing in some way... Anyway this concept needs to be modified further then to be accepted by all, but idk how.